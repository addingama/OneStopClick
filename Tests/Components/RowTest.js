import 'react-native'
import React from 'react'
import Row from '../../App/Components/Row'
import renderer from 'react-test-renderer'

test('Row component renders correctly', () => {
  const tree = renderer.create(<Row />).toJSON()
  expect(tree).toMatchSnapshot()
})


// import React from 'react'
// import { Text } from 'react-native'
// import { shallow } from 'enzyme'
// import toJson from 'enzyme-to-json'
// import Row from '../../App/Components/Row'

// describe('Row Component', () => {
//   it('should render without issues', () => {
//     const component = shallow(<Row />)

//     expect(component.length).toBe(1)
//     expect(toJson(component)).toMatchSnapshot()
//   })

//   it('should render children', () => {
//     const component = shallow(
//       <Row containerStyle={{ height: 100 }}><Text>Hi</Text></Row>
//     )

//     expect(component.find('Text').length).toBe(1)
//   })

//   it('should render with onPress', () => {
//     const onPress = jest.fn()
//     const component = shallow(
//       <Row
//         onPress={onPress}
//         containerStyle={{ backgroundRowor: 'peru' }}
//         size={3}
//       />
//     )

//     component.simulate('press')
//     expect(onPress).toHaveBeenCalled()
//   })
// })
