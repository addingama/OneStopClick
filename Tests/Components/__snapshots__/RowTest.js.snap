// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row component renders correctly 1`] = `
<View
  activeOpacity={1}
  style={
    Array [
      Object {
        "flex": 1,
        "flexDirection": "row",
      },
      undefined,
    ]
  }
/>
`;
